pipeline/bls.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python3
+"""
+This is a script to check BLS related information for a test network.
+
+Example usage:
+    python3 bls.py diff --shard 0
+"""
+
+
+import json
+import argparse
+import os
+import math
+
+from pyhmy import (
+    util
+)
+import requests
+
+from fund import (
+    get_network_config,
+    get_endpoints,
+)
+
+env = os.environ.copy()
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Network BLS information')
+    parser.add_argument("command", help="What BLS key function to do on network. Supported commands:\n\n"
+                                        "\t`ref`  \tGet reference/configured BLS keys per node for network.\n\n"
+                                        "\t`dist` \tGet current distribution of bls keys with IP address.\n\n"
+                                        "\t`diff` \tGet missing BLS keys per node for network.\n\n")
+    parser.add_argument("--shard", default=None, help="Specify a shard for a command", type=int)
+    parser.add_argument("--raw", action="store_true", help="Disable pretty printing")
```

Comment:
I'd like to recommend this module (https://docs.python.org/3/library/pprint.html) to print out a nested data structure if necessary.